third_party/opa/data-and-policies/meshfordata-policy-lib/helper_functions.rego

```diff
@@ -40,8 +40,8 @@ column_has_tag(tag) {
 	compare_str(tag, input.details.metadata.components_metadata[_].tags[_])
 }
 
-check_purpose(purpose) {
-    compare_str(purpose, Purpose())
+check_intent(intent) {
+    compare_str(intent, Intent())
 }
```

Comment:
Why do we have each rego file duplicated?

Reply:
Actually many files in third_party/opa needs to be removed (as part of https://github.com/IBM/the-mesh-for-data/issues/510). I will raise another PR for doing this. These changes are mainly to keep consistency for now. Hope it is ok. Thanks.